### ISSUE TITLE
Default to use the same browser for 'open' as 'run'

### ISSUE THREAD

**Issue by Graham42** [cypress-io/cypress#860]
- Operating System: Ubuntu 16.04
- Cypress Version: 1.0.3
- Browser Version: 
    - Chrome 62.0.3202.75 
    - whatever version of Electron comes with Cypress 1.0.3

### Is this a Feature or Bug?
Bug

### Current behavior:
Tests that pass when running cypress with `open` can fail when using `run` due to browser differences.

### Desired behavior:
Using default settings, if all tests pass in the live watch version (`open`) they should all pass when using `run`.

This can be solved by using the same browser for both so that issues can be easily debugged. Ideally use Chrome because that's more representative of the web than Electron. If the goal is to have a packaged browser with cypress, could look into [puppeteer](https://github.com/GoogleChrome/puppeteer)

### How to reproduce:
Create a test for an app that uses the history api (ie uses react-router with Redirect component)

**Comment by brian-mann** [comment]
Semi duplicate of https://github.com/cypress-io/cypress/issues/832

The problem here isn't the local environment (that is easily solvable) as you suggested. We already cache the last browser you opened and could just automatically start that one when running from the CLI.

The problem is that in CI - you would have to take many additional steps to have Chrome installed.

What I proposed in the other issue is to use Chrome by default but then open Electron as a fallback (and print the browser to the console). It's a little more magical but does pair up better to what users want.

We actually **do** do the same thing as puppeteer - we embed Electron so that every single user of Cypress only ever needs to install Cypress to at least start writing tests.

We could upgrade Electron from chromium 53 to 58 now, but we've been waiting for them to release another major update to node 8.x.x which will then bump up chromium to be pretty much on par with Chrome.
